idd/Energy+.idd.in

```diff
@@ -71628,6 +71628,125 @@ Boiler:Steam,
        \retaincase
        \default General
 
+Chiller:Electric:ASHRAE205,
+   \min-fields 12
```

Comment:
You have min fields set to 12 here.  The 12th field is _Chilled Water Maximum Requested Flow Rate_, which does not have a `\required-field` attribute.  So is the min fields 11?  Or is the 12th field required?

Reply:
(please address this one)

Reply:
Fixed (min=11).